client/components/plans/plan-price/index.jsx

```diff
@@ -20,6 +25,11 @@ module.exports = React.createClass( {
 				return this.translate( 'Free', { context: 'Zero cost product price' } );
 			}
 
+			if ( abtest( 'monthlyPlanPricing' ) === 'monthly' && this.props.isInSignup ) {
+				const monthlyPrice = +( rawPrice / 12 ).toFixed( 2 );
+				formattedPrice = formattedPrice.replace( rawPrice, monthlyPrice );
```

Comment:
Good point, we keep this in mind going down the line when implementing the winning variation. For the sake of the test, I don't see this as a blocker.